tutordiscovery/plugin.py

```diff
@@ -95,7 +96,7 @@
 
 
 # Automount /openedx/discovery folder from the container
-@tutor_hooks.Filters.COMPOSE_MOUNTS.add()
+@tutor_hooks.Filters.COMPOSE_MOUNTS.add()  # type: ignore
```

Comment:
Nope, let's not type-ignore this (same below).

Reply:
actually this was needed to run pylint checks which were failing before.

Reply:
Your "ignore" statement is not for pylint but for mypy. Mypy checks are failing probably because tutor is installed in editable mode: https://github.com/overhangio/tutor/issues/956
These tests should not fail in CI.

Reply:
ohk my bad, had to say mypy :)
If that's the case, I will remove it now.